commit 45a8d7c367300d3003838a4e0241915cf89fe3f9 (refs/changes/78/4878/3)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-09-25T20:26:27+00:00 (1 year ago)
    
    TRITON-786 Update schema for HVM API
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/package.json b/package.json
index 932f70e..48ef53f 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "assert-plus": "~1.0.0",
     "bunyan": "1.8.5",
     "libuuid": "0.2.1",
-    "joyent-schemas": "git+https://github.com/joyent/schemas.git#7af0f69e0ce0e589c8acd33b642861f54785b746",
+    "joyent-schemas": "git+https://github.com/joyent/schemas.git#9f0791091868f449c8933902b556652e6c24875c",
     "jsprim": "1.3.1",
     "vasync": "~1.6.4"
   },
